src/consensus/consensus.h

```diff
@@ -6,11 +6,38 @@
 #ifndef BITCOIN_CONSENSUS_CONSENSUS_H
 #define BITCOIN_CONSENSUS_CONSENSUS_H
 
+#include "consensus/params.h"
+
+#include <stdint.h>
+
+class CTransaction;
+class CValidationState;
+
+/**
+ * Consensus validations:
+ * Check_ means checking everything possible with the data provided.
+ * Verify_ means all data provided was enough for this level and its "consensus-verified".
+ */
+namespace Consensus {
+
+/** Transaction validation functions */
+
+/**
+ * Context-independent CTransaction validity checks
+ */
+bool CheckTx(const CTransaction& tx, CValidationState& state, uint64_t nMaxTxSize);
```

Comment:
Why not just pass `consensusParams` as you have done for the other two functions?
Might as well be consistent.